web/models.py

```
@@ -2804,3 +2804,193 @@ class ScheduledPost(models.Model):
+
+
+class ChapterResource(models.Model):
+    """Resources shared within chapters."""
+    
+    RESOURCE_TYPES = [
+        ('document', 'Document'),
+        ('template', 'Template'),
+        ('guide', 'Guide'),
+        ('tool', 'Tool'),
+        ('other', 'Other'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)